crates/optimism/node/src/node.rs

```diff
@@ -747,32 +744,46 @@ where
 
 /// A basic optimism network builder.
 #[derive(Debug, Default, Clone)]
```

Comment:
let's impl Clone manually because, the derive restricts this to `PooledTx: Clone` etc.